api/features/permissions.py

```diff
@@ -185,3 +186,46 @@ def has_permission(self, request, view):
             permission=MANAGE_SEGMENT_OVERRIDES,
             environment=environment,
         )
+
+
+class FeatureImportPermissions(IsAuthenticated):
+    def has_permission(
+        self, request: Request, view: "WrappedAPIView"  # noqa: F821
```

Comment:
Here (and in a few other places) the `WrappedAPIView` is the actual type but it isn't importable from Django Rest Framework because they generate the type in a weird scope (in a decorator, iirc).